lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -161,6 +171,9 @@ def configure(conf)
       if (@rename_time || @rename_time_if_not_exist) && @use_undefined && !@keep_fields.key?(@src_time_name)
         raise Fluent::ConfigError, "Field [#{@src_time_name}] must be listed in default_keep_fields or extra_keep_fields"
       end
+      if @undefined_dot_replace_char == 'UNUSED'
+        @undefined_dot_replace_char = nil
```

Comment:
style more then anything but you could say:

`@undefined_dot_replace_char = nil if @undefined_dot_replace_char == 'UNUSED'`

Reply:
done